### PR TITLE
Zigbee2mqtt add missing exposes owon pir313 e

### PR DESCRIPTION
**Pull request by William-De71**
### Pull Request check-list

To ensure your Pull Request can be accepted as fast as possible, make sure to review and check all of these items:

- [ ] If your changes affects code, did your write the tests?
- [ ] Are tests passing? (`npm test` on both front/server)
- [ ] Is the linter passing? (`npm run eslint` on both front/server)
- [ ] Did you run prettier? (`npm run prettier` on both front/server)
- [ ] If you are adding a new features/services, did you run integration comparator? (`npm run compare-translations` on front)
- [ ] Did you test this pull request in real life? With real devices? If this development is a big feature or a new service, we recommend that you provide a Docker image to the community ([french forum](https://community.gladysassistant.com/)/[english forum](https://en-community.gladysassistant.com/)) for testing before merging.
- [ ] If your changes modify the API (REST or Node.js), did you modify the API documentation? (Documentation is based on comments in code)
- [ ] If you are adding a new features/services which needs explanation, did you modify the user documentation? See [the GitHub repo](https://github.com/GladysAssistant/v4-website) and the [website](https://gladysassistant.com).
- [ ] Did you add fake requests data for the demo mode (`front/src/config/demo.js`) so that the demo website is working without a backend? (if needed) See [https://demo.gladysassistant.com](https://demo.gladysassistant.com).

NOTE: these things are not required to open a PR and can be done afterwards / while the PR is open.

### Description of change

Add tamper and battery low expose in zigbee2mqtt integration

image

### CODE REVIEW

File: server/services/zigbee2mqtt/exposes/binaryType.js [hunk 1]
```diff
@@ -78,6 +78,18 @@ const names = {
       type: DEVICE_FEATURE_TYPES.SENSOR.BINARY,
     },
   },
+  battery_low: {
+    feature: {
+      category: DEVICE_FEATURE_CATEGORIES.BATTERY,
+      type: DEVICE_FEATURE_TYPES.SENSOR.BINARY,
```
Comment (Pierre-Gilles) [inline server/services/zigbee2mqtt/exposes/binaryType.js:84]: Same feedback as the other PR, do it makes sense to categorize the feature as a battery feature, where it's not?

Imagine we have a device that have a feature "battery_high": true/false, how do we handle those features?

I think we need another category for this type of sensor

BATTERY_LOW seems good!

**Review by Pierre-Gilles — CHANGES_REQUESTED** [review]
Thanks for the PR! :)

File: server/services/zigbee2mqtt/exposes/binaryType.js (hunk 1 above)
Comment (William-De71) [reply server/services/zigbee2mqtt/exposes/binaryType.js:84]: You are right, modification done!

**Review by Pierre-Gilles — APPROVED** [review]
Looks good to me! Tested locally, thanks for the PR :)

**Pull request merged**